docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -36,7 +38,7 @@ MariaDB is a generally-compatible fork of MySQL. Where you see a reference to "M
 {{< /note >}}
 
 {{< note >}}
-The steps in this guide require root privileges. Be sure to run the steps below as `root` or with the `sudo` prefix. For more information on privileges, see our [Users and Groups](/docs/tools-reference/linux-users-and-groups/) guide.
+The steps in this guide require root privileges, and commands are run with `sudo` unless otherwise noted. For more information on privileges, see our [Users and Groups](/docs/tools-reference/linux-users-and-groups/) guide.
```

Comment:
This change has some impacts further down the guide; I wanted to be explicit about the assumptions we're making for the level of permission that the Linux user has. In general, we write guides from the perspective of a limited Linux user that has sudo privileges and test the commands in the guide against that assumption.

The only place in this guide that I thought was an exception to this was the 'aliases' section, which I'll add more details about below